examples/resources/git_repository/resource_cluster.tf

```diff
@@ -0,0 +1,32 @@
+# Create Tanzu Mission Control git repository with attached set as default value.
+resource "tanzu-mission-control_git_repository" "create_git_repository" {
```

Comment:
Differentiate on the resource names based on scope.

Reply:
updated PTAL